src/content/en/tools/workbox/guides/_shared/generate-sw-string-schema.html

```diff
@@ -13,7 +13,9 @@
     <p>
       This only needs to be set if you want to precache local files matching <code>glob</code>
       patterns. It does not need to be set if you have <code>self.__precacheManifest</code> defined
-      via a file included in <code>importScripts</code>.
+      via a file included in <code>importScripts</code>. In other words, developers using the
```

Comment:
I find this section confusing and I feel it could be more direct.

"This option sets the base you wish to match `globPatterns` against, relative to the work directory. If you don't want to precache files then this configuraiton isn't needed."

Regarding this new piece, I would assume this would feed into self.__precacheManifest (i.e. the files would be added to that list, if thats the case then why isn't being mentioned here as a reason not to use it? and if it's not added to that list, why not?

Reply:
I've moved this to its own webpack-specific include file, so the language can be more precise.

It now reads "This only needs to be set if you want to precache additional local files that are not created by the webpack build process."